application/config/config.go

```diff
@@ -626,6 +629,13 @@ func (c *Config) SetManageBinariesAutomatically(enabled bool) {
 	c.manageBinariesAutomatically.Set(enabled)
 }
 
+func (c *Config) ManageCliBinariesAutomatically() bool {
+	if c.engine.GetConfiguration().GetString(cli_constants.EXECUTION_MODE_KEY) == cli_constants.EXECUTION_MODE_VALUE_EXTENSION {
```

Comment:
Why do we need `EXECUTION_MODE_VALUE_EXTENSION? Can't we use `EXECUTION_MODE_VALUE_STANDALONE`? The LS doesn't need to know if there is an extension mode.